gr-blocks/lib/packed_to_unpacked_impl.cc

```diff
@@ -62,19 +62,33 @@ void packed_to_unpacked_impl<T>::forecast(int noutput_items,
     }
 }
 
+template <class T>
+unsigned int packed_to_unpacked_impl<T>::log2_l_type()
+{
+    unsigned int val = sizeof(T);
```

Comment:
I would have preferred 
```suggestion
    unsigned int val = sizeof(T) * 8;
```
instead of the + 3 below, but I'll classify that as a matter of taste.